pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -715,6 +715,13 @@ def test_to_datetime_unprocessable_input(self):
         )
         self.assertRaises(TypeError, to_datetime, [1, '1'], errors='raise')
 
+    def test_to_datetime_numerical_input(self):
+        self.assertRaises(ValueError, to_datetime, arg=int(1000))
```

Comment:
you can also do this with parametrize: http://pandas-docs.github.io/pandas-docs-travis/contributing.html#how-to-use-parametrize